src/Microsoft.IdentityModel.Protocols/Configuration/HttpDocumentRetriever.cs

```diff
@@ -75,6 +76,18 @@ public HttpDocumentRetriever(HttpClient httpClient)
         /// </summary>
         public bool RequireHttps { get; set; } = true;
 
+        /// <summary>
+        /// If set specifies the protocol version to use when sending HTTP requests.
+        /// </summary>
+        public Version HttpVersion { get; set; }
```

Comment:
do we need more documentation on this? ex in the wiki?

Reply:
I could create a separate PR for the Wiki and add a page explaining how to enable HTTP/2. What do you think?

Reply:
whichever works for you. We can also take the content in a comment here and copy it over for you

Reply:
I don't think we can create a PR for a wiki?

If you paste the content here, I can create a wiki page for it.